app/src/main/kotlin/com/wire/android/ui/common/SearchBar.kt

```diff
@@ -75,16 +95,26 @@ fun SearchBarUI(placeholderText: String, modifier: Modifier = Modifier, onTextTy
                 }
             }
         },
-        textStyle = LocalTextStyle.current.copy(fontSize = 14.sp, textAlign = TextAlign.Start),
-        placeholderTextStyle = LocalTextStyle.current.copy(fontSize = 14.sp, textAlign = TextAlign.Center),
+        interactionSource = interactionSource,
+        textStyle = textStyle.copy(fontSize = 14.sp),
+        placeholderTextStyle = placeholderTextStyle.copy(fontSize = 14.sp),
         placeholderText = placeholderText,
         maxLines = 1,
         singleLine = true,
     )
 }
 
+@Composable
+fun rememberSearchInputState(): SearchInputState = remember { SearchInputState() }
+
+class SearchInputState {
```

Comment:
Do you think it's possible to have this class marked as private? I think, I didn't see any other usages